src/Components/Results/Header/Header.tsx

```diff
@@ -15,8 +18,30 @@ const Buttons = () => {
 
 const ProgramsHeader = () => {
   const { programs } = useResultsContext();
+  const { theme } = useContext(Context);
+  const taxCredit = calculateTotalValue(programs, 'Tax Credit');
 
-  return <div>{programs.length}</div>;
+  const estimatedMonthlySavings = programs.reduce(
+    (eachEstimatedMonthlySavings, program) => eachEstimatedMonthlySavings + program.estimated_value,
+    0,
+  );
+
+  return (
+    <CardContent sx={{ backgroundColor: theme.secondaryBackgroundColor }}>
+      <div className="results-header">
+        <div className="results-header-programs-count-text">
+          <div className="results-header-programs-count">{programs.length}</div>
+          <div>Programs Found</div>
+        </div>
+        <div style={{ marginLeft: '15px' }}>
+          <div className="results-header-values">${Math.round(estimatedMonthlySavings / 12).toLocaleString()}</div>
+          <div className="results-header-label">Estimated Monthly Savings</div>
+          <div className="results-header-values">${taxCredit}</div>
+          <div className="results-header-label">Estimated Tax Credit</div>
+        </div>
```

Comment:
What do you think about using more semantic html tags for some of these elements? We want to make sure that we're making our website as accessible as we possibly can by using semantic html tags.

Reply:
@ivonne-hernandez  Please check the updated code, let me know if you have something specific what would you like to substitute with